distro/configuration/patientidentifiertypes/retire-core-identifiers.csv

```diff
@@ -0,0 +1,4 @@
+Uuid,Void/Retire,Name,display:en,display:km,Description,Required,Format,Format description,Validator,Location behavior,Uniqueness behavior
+,TRUE,Old Identification Number,,,,,,,,,
+,TRUE,Legacy ID,,,,,,,,,
```

Comment:
So, since this one is created from the other file, removing it from this other file will make sure we don't need to retire it here.

Reply:
The requested changes have been made. `"Legacy ID"` has been removed